src/data/__tests__/calculate.test.ts

```diff
@@ -16,51 +21,108 @@ const calcValue = (data: CalculatorData) => {
   return result.expectedValue
 }
 
-describe('calculate', () => {
-  // Prevailance is .1% with 6x underreporting factor
-  const baseTestData = {
-    riskBudget: BUDGET_ONE_PERCENT,
-    subLocation: 'mock city',
-    topLocation: 'mock state',
-    label: 'mock city',
-    population: '1,000,000',
-    casesPastWeek: 999, // will add 1 in pseudocount
-    casesIncreasingPercentage: 0,
-    positiveCasePercentage: 1,
-  }
+const dateAfterDay0 = (daysAfterDay0: number) => {
+  const date = new Date()
+  date.setTime(DAY_0.getTime() + daysAfterDay0 * MS_PER_DAY)
+  return date
+}
+
+// Prevailance is RATE (2x prevalance ratio)
+const baseTestData = {
+  riskBudget: BUDGET_ONE_PERCENT,
+  subLocation: 'mock city',
+  topLocation: 'mock state',
+  population: '1,000,000',
+  casesPastWeek: 2999, // will add 1 in pseudocount
+  casesIncreasingPercentage: 0,
+  positiveCasePercentage: 0,
+  prevalanceDataDate: dateAfterDay0(25), // prevalance ratio = 25 * positivity_rate ** 0.5 + 2
+}
 
-  // Variables that should be ignored for repeated/partner interactions.
-  const repeatedDontCare = {
-    setting: 'indoor',
-    distance: 'sixFt',
-    duration: 60,
-    theirMask: 'none',
-    yourMask: 'none',
-    voice: 'normal',
+// Variables that should be ignored for repeated/partner interactions.
+const repeatedDontCare = {
+  setting: 'indoor',
+  distance: 'sixFt',
+  duration: 60,
+  theirMask: 'none',
+  yourMask: 'none',
+  voice: 'normal',
+}
+
+const testData: (partial: Partial<CalculatorData>) => CalculatorData = (
+  partial,
+) => {
+  return {
+    ...defaultValues,
+    ...baseTestData,
+    ...partial,
   }
+}
 
-  const expectedPrevalance = 0.006
+describe('calculateLocationPersonAverage', () => {
+  it.each`
+    positiveCasePercentage | result
+    ${0}                   | ${RATE}
+    ${4}                   | ${BASE_RATE * (0.04 ** 0.5 * 25 + 2)}
+    ${6}                   | ${BASE_RATE * (0.06 ** 0.5 * 25 + 2)}
+    ${16}                  | ${BASE_RATE * (0.16 ** 0.5 * 25 + 2)}
+    ${100}                 | ${BASE_RATE * (25 + 2)}
+    ${null}                | ${BASE_RATE * (25 + 2)}
```

Comment:
Oh did you just add this from my slack comment or was this here before?

Reply:
It was here before. Maybe you were thinking of casesIncreasingPercentage?

Reply:
I must have been looking at that set of tests. But I don't think it matters as much there, since there's no point in the logic outlined for what to do with a null case there, right?

Reply:
So I'll call this resolved.

Reply:
In calculate.tsx, casesIncreasingPercentage is of type number, so it won't ever* be null.
positivityRate is `number?`, so it can be null.

Attempting to pass in `null` in the other test would cause the test to not compile. Putting `null` into the prevalence data would cause the build to fail.

Strong typing is good.

* you can totally make it null using unsafe casting (`as number`)